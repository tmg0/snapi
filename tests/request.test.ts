import { describe, it, expect } from 'vitest'
import { ofetch } from 'ofetch'

describe('request', () => {
  it('ungh h3', async () => {
    const data = await ofetch('https://ungh.cc/repos/unjs/h3')
    expect(data).toMatchSnapshot()
  })
})
