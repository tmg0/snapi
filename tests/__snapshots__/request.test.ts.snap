// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`request > ungh h3 1`] = `
{
  "repo": {
    "createdAt": "2020-11-17T14:15:44Z",
    "defaultBranch": "main",
    "description": "⚡️ Minimal H(TTP) framework built for high performance and portability ",
    "forks": 163,
    "id": 313641207,
    "name": "h3",
    "pushedAt": "2024-03-23T15:55:42Z",
    "repo": "unjs/h3",
    "stars": 3044,
    "updatedAt": "2024-03-26T20:10:56Z",
    "watchers": 15,
  },
}
`;
